api.planx.uk/send/uniform.test.ts

```diff
@@ -113,45 +137,19 @@ describe("createUniformSubmissionZip", () => {
   });
 
   test("the location plan is added to zip", async () => {
-    const geojson = {
-      type: "Feature",
-      geometry: {
-        type: "Polygon",
-        coordinates: [
-          [
-            [-0.07626448954420499, 51.48571252157308],
-            [-0.0762916416717913, 51.48561932090584],
-            [-0.07614058275089933, 51.485617225458554],
-            [-0.07611118911905082, 51.4857099488319],
-            [-0.07626448954420499, 51.48571252157308],
-          ],
-        ],
-      },
-      properties: null,
-    };
     const payload = {
       sessionId: "1234",
-      passport: {
-        data: {
-          "property.boundary.site": geojson,
-        },
-      },
-      csv: [],
-      files: [],
     };
-    await createUniformSubmissionZip(payload);
+    await createUniformSubmissionZip(payload.sessionId);
     expect(mockAddLocalFile).toHaveBeenCalledWith("LocationPlan.htm");
     expect(mockWriteZip).toHaveBeenCalledTimes(1);
   });
 
   test("geojson and location plan is excluded when not present", async () => {
     const payload = {
-      sessionId: "1234",
-      passport: { data: {} },
-      csv: [],
-      files: [],
+      sessionId: "noGeoJSON",
```

Comment:
I think a cleaner / clearer solution here is as follows.

- Make `getSessionById` it's own `jest.fn()` at the top of this file that always returns a promise of `mockLowcalSession`

```ts
const mockGetSessionById = jest
  .fn()
  .mockResolvedValue(mockLowcalSession);

jest.mock("../client", () => ({
  _admin: {
    getSessionById: () => mockGetSessionById(),
    ...
  }
})
```

- When we hit a test that has different behaviour, explicitly just overwrite the mock here inside this one test

```ts
const lowcalSessionWithoutBoundary: { ...mockLowcalSession };
delete lowcalSessionWithoutBoundary.data.passport.data["property.boundary.site"];

mockGetSessionById.mockResolvedValue(lowcalSessionWithoutBoundary);
```

This means that the logic is a little closer to the test instead of separated at the top of the file and can be more easily expanded on a test-by-test basis than just adding to the mock created at the top of the file 👍 

An example of something similar would be how `mockGenerateOneAppXML()` is being used in this file - it gives different responses in different tests.